crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -2043,6 +2043,89 @@ impl OlmMachine {
         DehydratedDevices { inner: self.to_owned() }
     }
 
+    /// Get the stored encryption settings for the given room, such as the
+    /// encryption algorithm or whether to encrypt only for trusted devices.
+    ///
+    /// These settings can be modified via [`OlmMachine::set_room_settings`].
+    pub async fn get_room_settings(&self, room_id: &RoomId) -> StoreResult<Option<RoomSettings>> {
```

Comment:
The `get_` prefix isn't really Rust idiomatic. I know it's present in the store API, let's not propagate this error :-]. Can you rename `get_room_settings` to `room_settings` please?